tests/stdlib/tjsonutils.nim

```diff
@@ -119,5 +119,95 @@ template fn() =
     testRoundtrip(Foo[int](t1: false, z2: 7)): """{"t1":false,"z2":7}"""
     # pending https://github.com/nim-lang/Nim/issues/14698, test with `type Foo[T] = ref object`
 
+  block hashSet:
+    testRoundtrip(HashSet[string]()): "[]"
+    testRoundtrip([""].toHashSet): """[""]"""
+    testRoundtrip(["one"].toHashSet): """["one"]"""
+
+    var s: HashSet[string]
+    fromJson(s, parseJson("""["one","two"]"""))
+    doAssert s == ["one", "two"].toHashSet
+    let jsonNode = toJson(s)
+    doAssert jsonNode.kind == JArray
```

Comment:
can you make this shorter using sort? (and `import algorithm`):
something like:
```nim
let jsonNode = toJson(s)
# skip the other checks, they're redundant since it.str would fail if kind != JString, etc
doAssert jsonNode.mapIt(it.str).sorted == @["one", "two"]
```

Reply:
Fixed.